asv/plugins/git.py

```diff
@@ -15,12 +15,19 @@
 
 class Git(Repo):
     dvcs = "git"
-    _default_branch = "master"
 
     def __init__(self, url, mirror_path):
         self._git = util.which("git")
         self._path = os.path.abspath(mirror_path)
         self._pulled = False
+        # default branch
+        try:
+            self._default_branch = self._run_git(['config',
+                                                  'init.defaultBranch',
+                                                  ], display_error=False,
```

Comment:
Without `display_error=False`, tests which capture `stderr` and `stdout` to test against reference strings fails.